lib/trenchboot.robot

```diff
@@ -0,0 +1,19 @@
+*** Settings ***
+Library     Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Resource    terminal.robot
+
+
+*** Keywords ***
+TrenchBoot Telnet Root Login
```

Comment:
Not sure why

```robot
Telnet.Read Until    login:
Telnet.Write    root
Telnet.Set Prompt    root@tb:~#
Telnet.Read Until Prompt
```

doesn't work. Log shows that telnet received expected string:

```
TRACE 	recv b'root@tb:~' 	
TRACE 	recv b'# '
```

Maybe it needs prompt to be read in one sitting (TRACE line)

Reply:
Though this works 

```robot
Telnet.Read Until    login:
Telnet.Write    root
Telnet.Set Prompt    root@tb:~#
Telnet.Write Bare   \n
Telnet.Read Until Prompt
```

Reply:
https://github.com/Dasharo/open-source-firmware-validation/compare/5163e1a22995279403dcc7f277ea309cd4ac79b3..f3c14936b5b14706e5efa3208f4544fe2ee65cd3#diff-4151aa87911e86ebfb90f0322239f177edd64ee7cd38cbe782f644a55341f1f0